sale_partner_address_restrict/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module restricts the addresses to be used in the sales order form, according to the partner selected.
```

Comment:
In which manner?

This should be improved as from module name, we don't understand how the restriction is applied.